sub/source/docs/introduction.rst

```diff
@@ -4,15 +4,36 @@
 Introduction to Sub
 ===================
 
-**UNDER CONSTRUCTION**
+Sub is an advanced open-source autopilot system for submersible ROVs (Remote Operated Vehicle) that supports multiple vehicle configurations. It offers a variety of operating modes from fully manual to fully autonomous.
 
+.. image:: ../images/sub-system.jpg
+    :target: ../_images/sub-system.jpg
```

Comment:
shouldnt the dotted line be going from the vehicle to the buoy, instead of from the buoy to the computer?

Reply:
shows two different tether schemes....tethered boy with RF to base(RC/telem) or hardwired cable....
so I guess its not intuitively obvious, as I thought....will add some text to image